docs/self-managed/setup/guides/identity-oracledb.md

```diff
@@ -0,0 +1,141 @@
+---
+id: identity-oracledb
+title: "Using OracleDB with Identity for OIDC"
+sidebar_label: "OracleDB with Identity for OIDC"
+description: "Configure Identity to use OracleDB with OIDC"
+---
+
+import Tabs from "@theme/Tabs";
+import TabItem from "@theme/TabItem";
+
+## Overview
+
+This guide is intended for customers those internal policies or compliance requirements prevent the use of PostgreSQL.
+
+## Scope
+
+This documentation focuses solely on components of Camunda 8 that can operate without PostgreSQL. Components such as Optimize and Web Modeler, which require PostgreSQL for proper functionality, are excluded from this guide and should be disabled in environments where PostgreSQL is not used.
+
+```yaml
+postgresql:
+  enabled: false
+identityKeycloak:
+  enabled: false
+identityPostgresql:
+  enabled: false
+webModeler:
+  enabled: false
+webModelerPostgresql:
+  enabled: false
+optimize:
+  enabled: false
+```
+
+## Versions
+
+We tested with the following versions
+
+| Camunda Version | Database Version       | Driver Version |
+| --------------- | ---------------------- | -------------- |
+| 8.6.13          | Oracle 19C (Preferred) | 21.3.0.0       |
```

Comment:
As its only tested with 8.6 - should we also only post it in the 8.6 guide or are we confident enough/ fine with this guide also being in 8.7 and 8.8?

Reply:
In 8.8 it doesn't matter, because Identity is merged into Zeebe.

For 8.7, it must still work, so it's a chicken and egg thing. I would say let's document it anyway, and start testing it right away. But I suspect it will mostly just work once @panagiotisgts has forward ported it.